packages/components/src/ListView/Items/__snapshots__/items.test.js.snap

```diff
@@ -7793,7 +7793,7 @@ exports[`Items should render with provided id 1`] = `
                                 type="checkbox"
                               />
                               <span
-                                className="tc-listview-item-label"
+                                className="tc-listview-item-label tc-listview-item-label"
```

Comment:
doesn't look like it always work fine

Reply:
Item's classnames is defined like that : `classNames(theme['tc-listview-item-label'], 'tc-listview-item-label');`

So yes, I think this is working as expected.

Reply:
As seen with @acateland, theme classes are now prefixed by `theme-`.